src/components/tooltip/CallNode.css

```diff
@@ -6,6 +6,8 @@
   display: grid;
   grid-gap: 2px;
   grid-template-columns: repeat(4, min-content);
+  border-bottom: 1px solid var(--grey-30);
+  padding-bottom: 5px;
```

Comment:
I'm not sure if this `padding-bottom` is necessary here. Why did you add it?

Reply:
![screenshot from 2019-03-05 17-21-36](https://user-images.githubusercontent.com/25532570/53803769-b4dddc00-3f6b-11e9-8fc5-fe2a1c63f319.png)

It was initially very overlapping with the text above it , so I thought it could be added.

![screenshot from 2019-03-05 17-18-28](https://user-images.githubusercontent.com/25532570/53803804-ddfe6c80-3f6b-11e9-88d8-f9c4d7db7a7e.png)

Reply:
Actually the `padding-bottom` should replace the `margin-top` in the lower component.

The previous layout was:

Higher component
Lower component's margin
Lower component's border
Lower component's padding

Now, with your change, we have:

Higher component
Higher component's padding
Higher component's border
Lower component's margin
Lower component's padding

As a result it's now unbalanced. That's why the padding you add should be the same as the lower component's margin you'll remove, and equal to the lower component's padding.

And I think this is 10px.

i hope it's clear!

Reply:
Yup, Got it. Thanks a lot for the insight.